sledgehammer/start-up.sh

```diff
@@ -150,8 +150,13 @@ echo "*.* @@${ADMIN_IP}" >> /etc/rsyslog.conf
 service $RSYSLOGSERVICE restart
 
 # Sometimes at this point network is not up yet, wait for it
-echo "Waiting for admin server ($ADMIN_IP) to be reachable; will wait up to 60 seconds..."
-ping -c 1 -w 60 $ADMIN_IP > /dev/null || {
+n=60
+echo "Waiting for admin server ($ADMIN_IP) to be reachable; will wait up to $n seconds..."
+while [[ $n -gt 0 ]] && ! ping -q -c 1 -w 1 $ADMIN_IP > /dev/null ; do
```

Comment:
Alternatively (which I recently started preferring because it reads more "normal") `while (( $n > 0 )) &&`.

Reply:
updated it. would even work without '$' but I left it in for readability